.github/workflows/tag.yml

```diff
@@ -14,14 +17,10 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v2
-        with:
-          submodules: true
-
-      - name: Fetch History
-        run: git fetch --prune --unshallow
 
       - name: Create Tag
-        run: make -j2 tag
-        env:
-          VERSION: ${{ github.event.inputs.version }}
-        
\ No newline at end of file
+        uses: negz/create-tag@v1
```

Comment:
Should this be `crossplane/create-tag@v1`?

Reply:
https://github.com/negz/create-tag :)

Reply:
Yeah, this action lives in my GitHub. It's pretty generally applicable so I was planning to just leave it there rather than relate it to Crossplane.